### PR TITLE
cleanUP eyeTracker shut down

### PR DESCRIPTION
**Pull request by marcobarilari**
Added this simple shutdown command. 

Alternative is to add on top of that the code to save the file (to be 100% safe to not lose any piece of data during whatever situation). In this case though, `cleanUp` will need `cfg` as input to retrieve the edf filename. 

LMK what you think about it.

### CODE REVIEW

File: cleanUp.m [hunk 1]
```diff
@@ -16,13 +16,18 @@ function cleanUp()
     % Screen Close All
     sca;
 
+    % Shut down connection with Eyelink
+    if cfg.eyeTracker.do
```
Comment (Remi-Gau) [inline cleanUp.m:20]: Actually it seems you already need a call to `cfg` there, so you could save though I am tempted to keep this function just for PURE clean up.

Do we get a crash if we try `Eyelink('shutdown')` when there is no eyetracker? If not we can jus skip the check on `cfg.eyeTracker.do`.

Tried on my octave here but it is complaining about mex files

Comment (marcobarilari) [reply cleanUp.m:20]: > Actually it seems you already need a call to cfg there, so you could save though I am tempted to keep this function just for PURE clean up.

I know I am not able to work early in the morning, I was still sleeping... ok to not save the file here, actually I don't think it makes sense cause being in the `catch` section the experiment has not been started yet so nothing to save

> Do we get a crash if we try Eyelink('shutdown') when there is no eyetracker? If not we can jus skip the check on cfg.eyeTracker.do.
> 
> Tried on my octave here but it is complaining about mex files

it crashes on mine as well, we can keep this on hold for a better solution, LMK

Comment (Remi-Gau) [reply cleanUp.m:20]: > > Tried on my octave here but it is complaining about mex files
> 
> it crashes on mine as well, we can keep this on hold for a better solution, LMK

We can put the call to this in a catch/try block as well so we are sure it won't cause us trouble.

```suggestion
try
    Eyelink('shutdown')
catch
end
```

Comment (marcobarilari) [reply cleanUp.m:20]: makes sense

**Pull request merged**